flybywiresim/aircraft
# fix: fix ETA of TO waypoint display on ND

* fix: ETA of TO waypoint is not displayed prior to takeoff and after landing in real aircraft
* doc: changelog






Fixes #8200 

## Summary of Changes

Fixes an issue where ETA of TO waypoint is visible on the ND prior to take-off and landing.
I decided to display it when the aircraft is detected to be off the ground and turn it off when it's on the ground (nose wheel pressure detected).
## Screenshots (if necessary)


On the ground before the flight plan entered
<img width="953" alt="Screenshot 2023-10-14 225607" src="https://github.com/flybywiresim/aircraft/assets/856035/ccb16ef6-4af5-49ab-ac6d-3cbdb708f141">
On the ground after the flight plan entered
<img width="939" alt="Screenshot 2023-10-14 225825" src="https://github.com/flybywiresim/aircraft/assets/856035/817df081-b827-4f8b-beaf-e33a05c21dcb">
Take-off before rotation
<img width="945" alt="Screenshot 2023-10-14 230305" src="https://github.com/flybywiresim/aircraft/assets/856035/4ad005bc-3c2b-4785-9be5-b6266c09bde5">
Take-off after rotation
<img width="928" alt="Screenshot 2023-10-14 230337" src="https://github.com/flybywiresim/aircraft/assets/856035/f58cad65-a410-4a5e-ba50-d84638bcc0a6">
Approach
<img width="940" alt="Screenshot 2023-10-15 000527" src="https://github.com/flybywiresim/aircraft/assets/856035/bc25872c-0a88-4664-a58e-1a54d2a6c6a0">
Roll-out with main gear on the ground
<img width="608" alt="Screenshot 2023-10-15 000725" src="https://github.com/flybywiresim/aircraft/assets/856035/e82a5819-8e7d-4608-8b58-939cec51cecd">
Roll-out with nose wheel on the ground
<img width="605" alt="Screenshot 2023-10-15 000751" src="https://github.com/flybywiresim/aircraft/assets/856035/8637ccb3-f912-4fff-8d78-dd4a9415f430">

## References

Take-off
https://www.youtube.com/watch?v=8DLYoj6S_lw&t=884s
Landing
https://youtu.be/rhI5XGwlnaI?t=269



## Additional context



Discord username (if different from GitHub):
alexm1987
## Testing instructions

1. Start Cold and Dark
2. Complete IRS alignment, observe PPOS is displayed on the ND but not ETA to TO and bearing. Before the change it was showing ETA 00:00 and bearing 000°.
3. Enter the flight plan, observe that there's still no ETA displayed on the ND, but bearing and distance are visible now.
4. Take-off as usual, as soon as GS (ground speed) goes above 100kts ETA becomes visible on the ND.
5. Upon landing when there's no more active legs in the flight plan left, ND switches to PPOS: ETA, bearing and distance are not visible. Another possibility if you manage to slow down to 100kts GS when the active leg is still active, ETA becomes unavailable.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
